pkg/logging/multihandler.go

```diff
@@ -0,0 +1,82 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package logging
+
+import (
+	"context"
+	"log/slog"
+
+	"github.com/cilium/cilium/pkg/lock"
+)
+
+func NewMultiSlogHandler(handler slog.Handler) *multiSlogHandler {
+	return &multiSlogHandler{
+		mu:       lock.RWMutex{},
+		handlers: []slog.Handler{handler},
+	}
+}
+
+type multiSlogHandler struct {
+	mu       lock.RWMutex
+	handlers []slog.Handler
+}
+
+func (i *multiSlogHandler) Enabled(ctx context.Context, level slog.Level) bool {
+	i.mu.RLock()
+	defer i.mu.RUnlock()
+	for _, h := range i.handlers {
+		return h.Enabled(ctx, level)
+	}
```

Comment:
```suggestion
	for _, h := range i.handlers {
		if h.Enabled(ctx, level) {
		    return true
		}
	}
```
I think this is what we want right? Otherwise we always return the first handler state.